core/src/env/srml/types.rs

```diff
@@ -34,24 +35,8 @@ impl EnvTypes for DefaultSrmlTypes {
     type Hash = self::Hash;
 }
 
-/// The default SRML address type.
-#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash, Encode, Decode)]
-pub struct Address([u8; 32]);
-
-impl From<[u8; 32]> for Address {
-    fn from(address: [u8; 32]) -> Address {
-        Address(address)
-    }
-}
-
-impl<'a> TryFrom<&'a [u8]> for Address {
-    type Error = TryFromSliceError;
-
-    fn try_from(bytes: &'a [u8]) -> Result<Address, TryFromSliceError> {
-        let address = <[u8; 32]>::try_from(bytes)?;
-        Ok(Address(address))
-    }
-}
+/// The default SRML address type
+pub type Address = node_runtime::Address;
 
 /// The default SRML balance type.
 pub type Balance = u64;
```

Comment:
Can you convert all pDSL types to `node-runtime` types? So also `Hash` and `Balance` etc.?